### ISSUE TITLE
Deploy Silver Cluster in SDN Compartment

### ISSUE THREAD

**Issue by stewartshea** [BCDevOps/OpenShift4-RollOut#37]
- [x] Build Master/Infra VMs
- [x] Deploy Temporary NetApp VMs while we await production physical #154

Once Hardware Arrives, cluster to be re-built using physical hosts.
- [x] Have DXC update switchports to new vlan tags

- [x] Install OCP 4 on nodes - Will require some trial and error to handle physicals vs vms in the build process (bonded networks, physical disks/raid)

**Comment by StevenBarre** [comment]
3 of the 8 hosts have been switched back into the OCP 3 network subnets.

The masters and infra VMs have been built.

**Comment by StevenBarre** [comment]
Moving to backlog as its a lower priority than the other tickets I'm working on

**Comment by tosazuwa** [comment]
Blocked pending OCP  4 Architecture update

**Comment by StevenBarre** [comment]
We no longer have physicals to deploy onto as they were moved back into the OCP3 cluster.

**Comment by mitovskaol** [comment]
The decision was made to use the HP switches that have been intended for the DR site in Calgary for the Silver cluster in Kamloops. When Arista switches, that have originally been intended for use in the Silver cluster, arrive (some time in August), they will be swaped with the HP switches. NOTE: Platform wide outage is expected during the switch swap. Need to prepare and communicate with the product teams.

Waiting for the HP switches to be shipped from Calgary to Kamloops. ETA is Wed June 10th.

**Comment by StevenBarre** [comment]
Moving to blocked waiting on SDN compartment.

**Comment by StevenBarre** [comment]
Lots of progress here after much trial and error with bonded interfaces.

https://console-openshift-console.apps.silver.devops.gov.bc.ca/

Still needs some testing to confirm everything is happy, but 90% done.

**Comment by StevenBarre** [comment]
Complete
